docs/source/module_configuration/kubernetes.rst

```diff
@@ -1,17 +1,13 @@
-.. _mod-k8s:
-
 Kubernetes
 ==========
 
-Kubernetes manifests can be deployed via Runway, offering an ideal way to
-handle core infrastructure-layer (e.g. shared ConfigMaps & Service Accounts)
-configuration of clusters. Perform the following steps to align your k8s
-directories with Runway's requirements & best practices.
-
 
 Part 1: Adding Kubernetes to Deployment
 ---------------------------------------
 
+.. I think i was able to translate everything over to the new structure pretty well except for maybe this section.
+.. Does anything in this section need to be covered more in one of the new sections?
```

Comment:
@troyready if you think this section is represented well enough after the restructure, I can delete the file and mark as ready for review.

Reply:
Seems good to me